### PR TITLE
dlna: move root descriptor xml template to the static assets

### PR DESCRIPTION
**Pull request by sretlawd**
Reduce binary size.

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [x] I have added tests for all changes in this PR if appropriate.
- [x] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: cmd/serve/dlna/data/data.go [hunk 1]
```diff
@@ -2,3 +2,35 @@
 // The "go:generate" directive compiles static assets by running assets_generate.go
 
 package data
+
+import (
+	"io/ioutil"
+	"text/template"
+
+	"github.com/ncw/rclone/fs"
```
Comment (ncw) [inline cmd/serve/dlna/data/data.go]: This should be "github.com/rclone/rclone/fs" - that is why the CI is failing.

Comment (sretlawd) [reply cmd/serve/dlna/data/data.go]: Ahhh that makes sense.  :-)

**Comment by ncw** [comment]
I just fixed up the goimports - assuming it builds I'll merge :-)

**Comment by ncw** [comment]
Looking good - will merge now - thank you :-)

**Pull request merged**